### PR TITLE
Custom vaccination

### PR DESCRIPTION
**Pull request by sanjaymhj**
Adding custom vaccination
set notification for custom vaccination
@laxmankhanal @anmol-lf @bishalg

### CODE REVIEW

File: Vaccination/ViewController/CustomVaccinationVC.swift [hunk 1]
```diff
@@ -9,6 +23,20 @@ class CustomVaccinationVC: UIViewController {
...
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.vaccinationDescription.layer.borderWidth = 1//vaccinationName.layer.borderWidth
+        self.vaccinationName.layer.borderWidth = 1
+        self.vaccinationTime.layer.borderWidth = 1
+        var netHex = 0xefefef
+        self.vaccinationDescription.layer.borderColor = UIColor(red:(netHex >> 16) & 0xff, green:(netHex >> 8) & 0xff, blue:netHex & 0xff).CGColor
+        self.vaccinationName.layer.borderColor = UIColor(red:(netHex >> 16) & 0xff, green:(netHex >> 8) & 0xff, blue:netHex & 0xff).CGColor
+        self.vaccinationTime.layer.borderColor = UIColor(red:(netHex >> 16) & 0xff, green:(netHex >> 8) & 0xff, blue:netHex & 0xff).CGColor
```
Comment (bishalg) [inline Vaccination/ViewController/CustomVaccinationVC.swift]: its better if we define Colors as constant and just use it in VC and Views

Comment (sanjaymhj) [inline Vaccination/ViewController/CustomVaccinationVC.swift]: :ok:

File: Vaccination/ViewController/CustomVaccinationVC.swift [hunk 2]
```diff
@@ -10,6 +10,24 @@ class CustomVaccinationVC: UIViewController {
...
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setBorder()
+    }
+
+    func setBorder() {
+        self.vaccinationDescription.layer.borderWidth = CVDimension.borderWidth
+        self.vaccinationName.layer.borderWidth = CVDimension.borderWidth
+        self.vaccinationTime.layer.borderWidth = CVDimension.borderWidth
+        var uiColor = UIColor(red: (CVDimension.borderColor >> 16) & 0xff, green: (CVDimension.borderColor >> 8) & 0xff, blue: CVDimension.borderColor & 0xff)
```
Comment (kusalshrestha) [inline Vaccination/ViewController/CustomVaccinationVC.swift]: find a better name inplace of uiColor.. like borderColor

Comment (sanjaymhj) [inline Vaccination/ViewController/CustomVaccinationVC.swift]: :ok:

**Pull request merged**